tsdb/head.go

```diff
@@ -179,6 +181,14 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 			Name: "prometheus_tsdb_checkpoint_creations_total",
 			Help: "Total number of checkpoint creations attempted.",
 		}),
+		labelPairs: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "prometheus_tsdb_head_label_pairs_total",
+			Help: "Total number of label pairs in the head",
+		}),
+		labelValues: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "prometheus_tsdb_head_label_values_total",
+			Help: "Total number of label values in the head",
```

Comment:
This is the size of the label values, not the number of label values (which would be the same as the number of label pairs).